internal/storage/config.go

```diff
@@ -139,6 +148,17 @@ func (s JSONConfigStore) Put(cfg Config) error {
 	return atomicWriteFile(filepath.Join(s.ConfigPath, configFileName), cfgBytes, 0o774)
 }
 
+func (s JSONConfigStore) Exists() (bool, error) {
+	_, err := os.Stat(filepath.Join(s.ConfigPath, configFileName))
+	if err != nil {
+		if os.IsNotExist(err) {
+			return false, nil
+		}
+		return false, err
+	}
+	return true, nil
```

Comment:
purely a style nit, but I'm used to seeing this written more like:
```suggestion
	if _, err := os.Stat(filepath.Join(s.ConfigPath, configFileName)); errors.Is(err, fs.ErrNotExist) {
	} else if err != nil {
	} else {
	}
```

also apparently `IsNotExist` is soft-deprecated in favor of `errors.Is`: https://github.com/golang/go/blob/master/src/os/error.go#L88-L89

Reply:
I dig